### PR TITLE
View Jobs Applied

### PR DESCRIPTION
**Pull request by DasDeveloper**
Added a summary of jobs applied for a user which is accessible from the navbar.
Added necessary css.
Added backend functionality.

### CODE REVIEW

File: src/client/src/css/viewJobsApplied.css
Comment (Aboudik1452) [inline src/client/src/css/viewJobsApplied.css:1]: CSS is broken when resizing the window to a width higher than 640 pixels. You can see the horizontal scroll bar
image

**Review by Aboudik1452 — CHANGES_REQUESTED** [review]
Good work overall.
Please take a look at my comments, and let me know if you have questions!

File: src/server/models/user.js [hunk 1]
```diff
@@ -32,6 +32,7 @@ const UserSchema = new mongoose.Schema({
         type: Number,
     },
     jobsApplied: [],
+    jobsAppliedObject:[],
```
[the remark this answers is not in the document]
Comment (DasDeveloper) [reply src/server/models/user.js:35]: The difference is that one the first one contains only the id's of the jobs he has applied for while the second contains all the jobs he has applied for in JSON format. Changing the initial attribute to objects was too complicated and caused errors for other pages which is why i chose to add another one instead.

File: src/server/routes/userRoutes.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ router.route("/newConnection").post(usersControllers.updateConnections);
 router.route("/deleteAwaiting").patch(usersControllers.deleteAwaitingConnections);
 //router to check if 
 router.route("/:id/jobsApplied").get(usersControllers.getUserJobsApplied);
+router.route("/:id/jobsObjectApplied").get(usersControllers.getUserJobsObjectApplied)
```
[the remark this answers is not in the document]
Comment (DasDeveloper) [reply src/server/routes/userRoutes.js:36]: Yes, both are necessary. I imagine it will also be useful for future sprints.

**Review by Aboudik1452 — APPROVED** [review]

**Review by Kunal22shah — APPROVED** [review]
LGTM

**Pull request merged**